notebooks/generative/fashionmnist_vae.livemd

```diff
@@ -0,0 +1,304 @@
+# A Variational Autoencoder for MNIST
+
+```elixir
+Mix.install([
+  {:exla, "~> 0.3.0"},
+  {:nx, "~> 0.3.0"},
+  {:axon, "~> 0.2.0"},
+  {:req, "~> 0.3.1"},
+  {:kino, "~> 0.6.2"},
+  {:scidata, "~> 0.1.9"},
+  {:stb_image, "~> 0.5.2"}
+])
+```
+
+## Introduction
+
+In this notebook, we'll be building a variational autoencoder (VAE). This will help demonstrate splitting up models, defining custom layers in Axon and a few additional Kino tricks for training models.
+
+This notebook builds on the [denoising autoencoder example](https://github.com/elixir-nx/axon/blob/main/notebooks/generative/mnist_autoencoder_using_kino.livemd) and turns the simple autoencoder into a variational one for the same dataset.
+
+## Training a simple autoencoder
+
+This section will proceed without much explanation as most of it is extracted from [denoising autoencoder example](https://github.com/elixir-nx/axon/blob/main/notebooks/generative/mnist_autoencoder_using_kino.livemd). If anything here doesn't make sense, take a look at that notebook for an explanation.
+
+```elixir
+defmodule Utils do
```

Comment:
Just a nit, but this all seems related to data, so maybe a better name is `defmodule Data` or something...

```suggestion
defmodule Data do
```